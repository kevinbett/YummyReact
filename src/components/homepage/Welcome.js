import React, {Component} from 'react';


class Welcome extends Component {
  state = {
    selectedIndex: 0,
  };

  select = (index) => this.setState({selectedIndex: index});

  render() {
    return (
      <div>
        <h1>Welcome</h1>
    </div>
    );
  }
}
export default Welcome;