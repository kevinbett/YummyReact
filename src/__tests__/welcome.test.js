import React from 'react';
import {shallow} from 'enzyme';
import {shallowToJson} from 'enzyme-to-json';

import Welcome from '../homepage/welcome';

describe('Welcome Component', () => {
    const wrapper = shallow(<Welcome />);
    it('renders without crashing', () => {
        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });
});