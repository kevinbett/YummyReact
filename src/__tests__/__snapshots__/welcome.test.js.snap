// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome Component renders without crashing 1`] = `
<div>
  <h1>
    Welcome
  </h1>
</div>
`;
